cxx/fbjni/detail/Meta.h

```diff
@@ -293,6 +293,9 @@ struct jtype_traits {
     return detail::SimpleFixedString<len>(Repr::get_instantiated_base_name());
   }
  public:
+  using descriptorType = decltype(jtype_traits<T>::descriptor());
+  using basenameType = decltype(jtype_traits<T>::base_name());
+
   // The jni type signature (described at
   // http://docs.oracle.com/javase/1.5.0/docs/guide/jni/spec/types.html).
   static constexpr decltype(jtype_traits<T>::descriptor()) /* detail::SimpleFixedString<_> */ kDescriptor = descriptor();
```

Comment:
Now that you have the typedefs, I think you should use them here and line 305, too.

Reply:
Thanks, @mhorowitz - this is done.